apps/settings/lib/SetupChecks/ForwardedForHeaders.php

```diff
@@ -59,6 +59,22 @@ public function run(): SetupResult {
 			return SetupResult::error($this->l10n->t('Your "trusted_proxies" setting is not correctly set, it should be an array.'));
 		}
 
+		foreach ($trustedProxies as $proxy) {
+			$addressParts = explode('/', $proxy, 2);
+			if (filter_var($addressParts[0], FILTER_VALIDATE_IP) === false) {
```

Comment:
```suggestion
			if (filter_var($addressParts[0], FILTER_VALIDATE_IP) === false || !ctype_digit($addressParts[1] ?? 24)) {
```
This would avoid duplicating the if inner part.

Reply:
Looks cleaner! Thank you for the input :)